### PR TITLE
ignition-transport2: new formula

### PR DESCRIPTION
**Pull request by scpeters**

### CODE REVIEW

File: ignition-transport2.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class IgnitionTransport2 < Formula
+  desc "Transport middleware for robotics"
+  homepage "http://ignitionrobotics.org"
+  url "https://bitbucket.org/ignitionrobotics/ign-transport/get/dfbb1c1.tar.gz"
```
Comment (caguero) [inline ignition-transport2.rb]: Is this URL correct? I see version 1.2.0 in the CMakeLists.txt inside dfbb1c1.tar.gz

Comment (scpeters) [inline ignition-transport2.rb]: you're right, it won't have the same package name. I'll change it to nate's branch

**Comment by caguero** [comment]
Is there any way to add an optional dependency or recommendation to a formula? I'm thinking on ign-tools.

File: ignition-transport2.rb (hunk 1 above)
Comment (scpeters) [inline ignition-transport2.rb]: I just updated the commit to download and squashed

**Comment by scpeters** [comment]
ign-tools is currently a required dependency. we can make it recommended (on by default) or optional (off by default)

**Pull request closed without merging**